samples/drivers/watchdog/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	aliases {
+		watchdog0 = &wdt010;
```

Comment:
nrf54l15 has this alias in the board definition, see:
https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/nordic/nrf54l15pdk/nrf54l15pdk_nrf54l15-common.dtsi#L63
@anangl: Shall I do the same for nrf54h20 cpuapp? What about cpurad (wdt010 instances are local and multiple cores can use wdt010 as it maps to a different base addresses)?

Reply:
Such alias is also crucial to run this sample
https://github.com/zephyrproject-rtos/zephyr/blob/main/samples/subsys/task_wdt/src/main.c#L24

Reply:
Yes, please add it in the board definition.

> What about cpurad (wdt010 instances are local and multiple cores can use wdt010 as it maps to a different base addresses)?

I'm not sure if I understand well what you are asking about, but I think it is okay to add same alias for cpurad.

Reply:
I moved watchdog0 alias to the board definition for Application core.

I decided to NOT define such alias for Radio core because:
- there are no other aliases defined on that core;
- I don't run samples/drivers/watchdog on Radio core; I don't know if such alias is needed by other samples/tests or just by this one.
- I don't want to pollute DTS with unused aliases. It can always be added later.